src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,44 @@
+.calendar {
+  width: ($cell-size * $colon-count)  + ($paddings * 2);
+
+  display: flex;
+  flex-wrap: wrap;
+  padding: $paddings;
+  gap: $gap;
+
+  @each $day, $numer in $week-day {
+    &--start--#{$day} &__day:nth-child(1) {
+      margin-left: ($cell-size * $numer) + ($gap * $numer);
```

Comment:
You can rewrite your formula like so: `($cell-size + $gap) * number` it will be the same